core/src/main/java/org/apache/calcite/sql/type/OperandTypes.java

```diff
@@ -408,6 +408,11 @@ private boolean hasFractionalPart(BigDecimal bd) {
       family(SqlTypeFamily.STRING, SqlTypeFamily.STRING,
           SqlTypeFamily.INTEGER, SqlTypeFamily.INTEGER);
 
+  public static final SqlSingleOperandTypeChecker STRING_STRING_INTEGER_INTEGER_STRING =
+      family(SqlTypeFamily.STRING, SqlTypeFamily.STRING,
+          SqlTypeFamily.INTEGER, SqlTypeFamily.INTEGER,
```

Comment:
This is not that necessary, you can new a fresh one directly where you need this.

Reply:
Sure. Will remove this. Thanks